app/src/main/java/com/kamron/pogoiv/clipboardlogic/tokens/UnicodeToken.java

```diff
@@ -14,9 +14,10 @@
  */
 
 public class UnicodeToken extends ClipboardToken {
-    String[] unicode_0_15 = {"⓪", "①", "②", "③", "④", "⑤", "⑥", "⑦", "⑧", "⑨", "⑩", "⑪", "⑫", "⑬", "⑭", "⑮"};
-    String[] unicode_0_15filled = {"⓿", "❶", "❷", "❸", "❹", "❺", "❻", "❼", "❽", "❾", "❿", "⓫", "⓬", "⓭", "⓮", "⓯"};
-    boolean filled;
+    private String[] unicode_0_15 = {"⓪", "①", "②", "③", "④", "⑤", "⑥", "⑦", "⑧", "⑨", "⑩", "⑪", "⑫", "⑬", "⑭", "⑮"};
+    private String[] unicode_0_15filled = {"⓿", "❶", "❷", "❸", "❹", "❺", "❻", "❼", "❽", "❾", "❿", "⓫", "⓬", "⓭", "⓮",
```

Comment:
This change (line 17-19) in unnecessary, please revert it (and remember to squash!)
:)